src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -57,9 +56,9 @@ public Errors validateFull(OverseasEntitySubmissionDto overseasEntitySubmissionD
 
     public Errors validatePartial(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
 
-        var entityNameDto = overseasEntitySubmissionDto.getEntityName();
-        if (Objects.nonNull(entityNameDto)) {
-            entityNameDtoValidator.validate(entityNameDto, errors, loggingContext);
+        var entityName = overseasEntitySubmissionDto.getEntityName();
+        if (StringUtils.isNotBlank(entityName)) {
```

Comment:
I think this need to be a 'non null' check. isNotBlank() will include an empty string which I think should then flag an error. Need a unit-test for this scenario...

Reply:
Separated null from empty check and just confined validate method to being an empty check as null has already been tested in both partial and full validation.